datarepo-clienttests/src/main/java/utils/KubernetesClientUtils.java

```diff
@@ -262,4 +288,24 @@ public static void waitForReplicaSetSizeChange(V1Deployment deployment, int numb
               + ")");
     }
   }
+
+  public static void modifyKubernetesPostDeployment(int podCount) throws Exception {
```

Comment:
Can you change this method name to something more specific to what it's modifying about the deployment? Say, changeReplicaSetSizeAndWait or something snappier : )

Reply:
Yes, good idea!